packages/fluent-ui/src/FieldTemplate/FieldTemplate.tsx

```diff
@@ -1,29 +1,40 @@
 import React from "react";
-import { FieldTemplateProps } from "@rjsf/utils";
+import { FieldTemplateProps, getTemplate, getUiOptions } from "@rjsf/utils";
 import { Text } from "@fluentui/react";
 
-const FieldTemplate = ({
-  id,
-  children,
-  errors,
-  help,
-  rawDescription,
-  classNames = "",
-  hidden,
-}: FieldTemplateProps) => {
+const FieldTemplate = (props: FieldTemplateProps) => {
+  const {
+    id,
+    children,
+    errors,
+    help,
+    rawDescription,
+    hidden,
+    uiSchema,
+    registry,
+  } = props;
+  const uiOptions = getUiOptions(uiSchema);
+  const WrapIfAdditionalTemplate = getTemplate(
```

Comment:
Adding the string name as the generic will ensure the proper typing
```suggestion
  const WrapIfAdditionalTemplate = getTemplate<"WrapIfAdditionalTemplate">(
```